platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,749 @@
+
+### Pipeline steps and computing resource requirements
+
+The nf-core/rnaseq pipeline involves several key steps, each with distinct computational requirements. Resource needs in this table are based on the `test_full` runs detailed previously:
+
+| **Pipeline step**                   | **Tools**                 | **Resource needs**           | **Description**                                                                                   |
+|-------------------------------------|---------------------------|------------------------------|---------------------------------------------------------------------------------------------------|
+| **Quality Control (QC)**            | FastQC, MultiQC           | Low-moderate CPU (50-200 % single-core usage), low memory (1-7 GB peak)     | Initial quality checks of raw reads to assess sequencing quality and identify potential issues.   |
+| **Read Trimming**                   | Trim Galore!              | High CPU (up to 700 % single-core usage), low memory (6 GB peak) | Removal of adapter sequences and low-quality bases to prepare reads for alignment.               |
+| **Read Alignment**                  | HISAT2, STAR              | Moderate-high CPU (480-600 % single-core usage), high memory (36 GB peak)       | Alignment of trimmed reads to a reference genome, typically the most resource-intensive step.    |
+| **Pseudoalignment**              | Salmon, Kallisto              | Moderate-high CPU (420 % single-core usage), moderate memory (18 GB peak)        | A faster, more accurate method of gene expression quantification than alignment using read compatibility.    |
+| **Quantification**                  | featureCounts, Salmon     | Moderate-high CPU (500-600 % single-core usage), moderate memory (18 GB peak) | Counting the number of reads mapped to each gene or transcript to measure expression levels.     |
+| **Differential Expression Analysis**| DESeq2, edgeR             | High CPU (650 % single-core usage), low memory (up to 2 GB peak )     | Statistical analysis to identify genes with significant changes in expression between conditions. |
+
+#### Overall run metrics
+
+The Fusion file system used with NVMe instance storage contributed to a 34 % improvement in total pipeline runtime and a 49 % reduction in CPU hours. 
+
+Total pipeline run cost: 
+- Fusion file system with fast instance storage: $ 34.90
+- Plain S3 storage without Fusion: $ 58.40
+
+![Run metrics overview](./_images/cpu-table.png)
+
+#### Process run time
+
+The Fusion file system demonstrates significant performance improvements for most processes in the nf-core/rnaseq pipeline, particularly for I/O-intensive tasks: 
+
+- The most time-consuming processes see improvements of 36-70%, saving hours of runtime in a full pipeline execution.
+- Most processes show significant performance improvements with Fusion, with time savings ranging from 35% to 99%.
```

Comment:
```suggestion
- The most time-consuming processes see improvements of 36-70%, saving hours of runtime in a full pipeline execution.
- Most processes show significant performance improvements with Fusion, with time savings ranging from 35% to 99%.
```
I'm not sure what the rule is here, but you have two decimal places on some numbers but not these, if they are referencing table numbers that are listed with 2 dp, I would say "approximately...", otherwise, I'd be specific and list the numbers.